web/src/components/__tests__/CityContentMenu.spec.tsx

```diff
@@ -97,4 +99,22 @@ describe('CityContentSidebar', () => {
 
     expect(getByText('layout:readAloud').parentElement?.parentElement).toHaveClass('Mui-disabled')
   })
+
+  it('should hide feedback on desktop', () => {
```

Comment:
```suggestion
  it('should hide feedback and PDF button on desktop', () => {
```

Reply:
This would be misleading, the pdf button is also shown on desktop but only for categories.